docs/beginner-checks.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started with Scorecard Checks for Supply Chain Security
+
+If you are an open-source developer, you know that maintaining the security posture of a project can feel like a full-time job, but it can be easier with a little guidance. [Scorecard](https://github.com/ossf/scorecard#openssf-scorecard), an automated tool created [by the Open Source Security Foundation](https://securityscorecards.dev/) (OpenSSF), checks projects for security best practices and scores projects on a scale between 1 and 10. Maintainers can use the Scorecard checks as guidelines to improve their project's security practices.
+
+Choosing which Scorecard checks to get started with can be overwhelming, though. On this page, we’ll walk you through getting started with some of the most important checks, focusing on the ones that give you the biggest payoff versus effort. We’ve broken them down into three categories based on stages of the development process: setting up your project, accepting contributions from others, and packaging the project to release to the world.
+
+## 1. Setting up your project
+
+The setup phase of an open-source project can be hectic, which can make it difficult to prioritize security concerns. Start your project off strong by focusing on Scorecard checks that help you secure your project dependencies and workflows.
+
+- Secure your dependencies with the Vulnerabilities check and the Dependency-Update-Tool check
+- Secure your workflows with the Token-Permissions check
+
+### The Vulnerabilities and Dependency-Update-Tool checks secure your dependencies
+
+Vulnerabilities are probably the most familiar security risk. By running Scorecard’s [Vulnerabilities check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#vulnerabilities), you’ll get information about known vulnerabilities in your project, either through your codebase or through your direct and (for most languages) indirect dependencies. Tracking these can be tricky, but it’s important: [95% of vulnerable dependencies are transitive](https://www.endorlabs.com/state-of-dependency-management).
+
+If vulnerabilities are found in your dependencies, there are a few options:
+
+- Update the dependency to a non-vulnerable version (if available)
+- Submit a patch to the vulnerable project
+- Replace the dependency with a non-vulnerable dependency
+- Remove the dependency and write code to take its place
+- If you are sure a vulnerability does not impact your project, you may ignore the dependency by creating an [osv-scanner.toml](https://google.github.io/osv-scanner/configuration/#ignore-vulnerabilities-by-id) file in the same directory as your dependency manifest
```

Comment:
I think Scorecard looks for it at the project's root by default

Reply:
Thanks! Changed the text to reflect that.